api/specs.proto

```diff
@@ -192,6 +192,9 @@ message ContainerSpec {
 	// SecretReference contains references to zero or more secrets that
 	// will be exposed to the container.
 	repeated SecretReference secrets = 12;
+
+	// Healthcheck describes how to check the container is healthy.
```

Comment:
The implication of a failing healthcheck is stronger in swarmkit. If the health check fails, we exit the task and reschedule elsewhere.